command_factory.py

```diff
@@ -0,0 +1,34 @@
+from functions import *
+
+
+class command_factory:
+    def __init__(self, args):
+        self.cmd = args.command.lower()
+        self.args = args.lower()
+
+    def execute(self):
```

Comment:
This function should create a command object.
Each command class has its own execute function.

factory design pattern is used to create an object not to execute it. That’s why the name  is ‘factory’